stable/database/values.yaml

```diff
@@ -430,7 +430,25 @@ database:
     #Some clusters require longer readiness probe timeouts
     readinessTimeoutSeconds: 5
 
+    # Table Partitions and Storage Groups (TPSG) is the mechanism by which a
+    # version of an SQL table row is stored only on a subset of Storage Managers
+    # (SMs). The "storageGroup" section configures the named Storage Group (SG)
+    # which the SMs installed with this Helm release will initially serve when
+    # their archives are created. The "UNPARTITIONED" storage group is
+    # implicitly added as it is served by every SM in the database. The
+    # information about served SGs is stored in the archive and may change over
+    # time. The SM will emit warning messages if there is a mismatch between the
+    # initially configured SG and the currently served SGs.
+    storageGroup: 
+      enabled: false
+      # The name of the storage group. Only alphanumeric and underscore ('_')
+      # characters are allowed. By default the Helm release name is used.
+      name: ""
+
   te:
+    # Provision Transaction Engine (TE) Deployment.
```

Comment:
A note about how this behaves with SGs / secondary deployment might be useful here, as well as / instead of in the helpers file where users are less likely to look.